pkg/metrics/server.go

```diff
@@ -34,11 +36,21 @@ func Serve(ctx context.Context, config Config) {
 		logrus.Fatalf("error creating the metrics listener: %v", err)
 	}
 
+	mux := http.NewServeMux()
```

Comment:
just out of curiosity, why did this get moved?

Reply:
Whoops, this was not meant to be committed. Let me revert this particular change 🙂

Reply:
Reverted.